python_modules/libraries/dagster-ssh/dagster_ssh/resources.py

```diff
@@ -11,10 +13,16 @@
     _check as check,
     resource,
 )
+from dagster._config.pythonic_config import ConfigurableResource
 from dagster._core.definitions.resource_definition import dagster_maintained_resource
+from dagster._core.execution.context.init import InitResourceContext
 from dagster._utils import mkdir_p
-from dagster._utils.merger import merge_dicts
+from paramiko.client import SSHClient
 from paramiko.config import SSH_PORT
+from pydantic import (
+    Field as PyField,
```

Comment:
I could be misremembering, but i thought we wanted to import `pydantic.Field` as plain `Field` and update `dagster.Field` to have a new import symbol?